### ISSUE TITLE
on a[href^="http:"], background-image dissappears

### ISSUE THREAD

**Issue by GoogleCodeExporter** [liudonghua123/ie7-js#121]
```
What steps will reproduce the problem?
1. html: <a href="http://www.google.com" title="google">google</a>
2. css: a[href^="http:"]{
    padding-left: 12px;
    background-image: url('img/icons/ext_link.gif');
    background-repeat: no-repeat;
    background-position: 0 0.45em;
}

What is the expected output? What do you see instead?
positioned background-image on external links.

What version of the product are you using? On what operating system?
http://ie7-js.googlecode.com/svn/version/2.0(beta3)/IE8.js
WinXP SP2 IE7

Please provide any additional information below.
works with "scripts disabled"

```

Original issue reported on code.google.com by `marine...@pixolution.ch` on 21 Oct 2008 at 9:02

**Comment by GoogleCodeExporter** [comment]
```
I can confirm the issue and say that the problem is also present with the rule
a[href*=something].
While the problem doesn't appear with a[href] or just a.
```

Original comment by `m.ep...@gmail.com` on 10 Jul 2009 at 12:18

**Comment by GoogleCodeExporter** [comment]
```
Another info:
it works with a[href=something] but not with a[href="something"]

So the problem seem to be with quotes...

And... Note that I'm having this problem on IE 7 but not on IE6 or IE8.
```

Original comment by `m.ep...@gmail.com` on 10 Jul 2009 at 12:22

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in version 2.1.
```

Original comment by `dean.edw...@gmail.com` on 11 Feb 2010 at 2:43
- Changed state: **Fixed**
